audit/projects/kubernetes-public/buckets/k8s-infra-tf-aws/iam.json

```diff
@@ -0,0 +1,24 @@
+{
+  "bindings": [
+    {
+      "members": [
+        "group:k8s-infra-aws-admins@kubernetes.io",
+        "projectEditor:kubernetes-public",
+        "projectOwner:kubernetes-public"
+      ],
+      "role": "roles/storage.legacyBucketOwner"
+    },
+    {
+      "members": [
+        "projectViewer:kubernetes-public"
+      ],
+      "role": "roles/storage.legacyBucketReader"
```

Comment:
Hm, this is not right, these buckets should be private: https://github.com/kubernetes/k8s.io/pull/1952

Reply:
I take this back.  The bucket itself can be read, but none of its contents.

I still plan on having a PR out that tightens access on these later, but can live with this permission in the meantime.